edge-functions/ab-testing-statsig/README.md

```diff
@@ -0,0 +1,41 @@
+# A/B testing with Statsig
+
+This example shows how to do A/B testing using statsig
+
+## Demo
+
+https://edge-ab-testing-statsig.vercel.app
+
+## How to Use
+
+You can choose from one of the following two methods to use this repository:
+
+### One-Click Deploy
+
+Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example):
+
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/vercel/examples/tree/main/edge-functions/ab-testing-statsig&project-name=ab-testing-statsig&repository-name=ab-testing-statsig)
```

Comment:
@goncy Can you add the requireved env variable here, and the env link to either this readme or to the place in statsig that they can use to get that env

https://vercel.com/docs/deploy-button